app/assets/javascripts/govuk-component/option-select.js

```diff
@@ -0,0 +1,136 @@
+(function ($) {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  function OptionSelect(options){
+    /* This JavaScript provides two functional enhancements to option-select components:
+      1) A link that clears all of the checkboxes (referred to here as the "clearing link")
+      2) Open/closing of the list of checkboxes - this is not provided for ie6 and 7 as the performance is too janky.
+    */
+
+    this.$optionSelect = options.el;
```

Comment:
since the prefix of $ means the variable is a jQuery object, shouldn't `options.el` be `options.$el`? And obviously the same naming when it's called on line 134